packages/app/src/Charcoal/App/AppContainer.php

```diff
@@ -2,40 +2,60 @@
 
 namespace Charcoal\App;
 
-// From Slim
-use Slim\Container;
-// From Pimple
-use Pimple\ServiceProviderInterface;
-// From 'charcoal-factory'
+use DI\Container;
+use DI\ContainerBuilder;
+use Psr\Container\ContainerInterface;
 use Charcoal\Factory\GenericFactory as Factory;
-// From 'charcoal-app'
-use Charcoal\App\AppConfig;
 use Charcoal\App\ServiceProvider\AppServiceProvider;
 
 /**
  * Charcoal App Container
  */
-class AppContainer extends Container
+class AppContainer implements ContainerInterface
 {
+    /**
+     * @var Container
+     */
+    private Container $container;
```

Comment:
It might be better to extend PHP-DI's `Container` instead of implementing PSR-11's interface and using PHP-DI internally.